### PR TITLE
ALCS-201: UX/UI Update

### PR DESCRIPTION
**Pull request by sandratoh**
## Various UI updates + small fixes:
* Paused card
* Date card
* Paused date card
* Header nav menu
* Application detail side nav
* Ng-select border colours
* Button transition
* Schedule card
* Kanban board header + column
* Comments

nav-dropdown-hover
date-card
side-nav-hover

### CODE REVIEW

File: alcs-frontend/src/app/features/board/board.component.scss [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
   height: 100%;
+  overflow: auto;
 }
 
 .board-header {
   display: flex;
   justify-content: space-between;
+  align-items: center;
   padding: 24px 80px 12px 76px;
+  position: sticky;
```
Comment (dhaselhan) [inline alcs-frontend/src/app/features/board/board.component.scss:15]: If you are setting sticky here, is it still needed on .column-header?

File: alcs-api/src/providers/typeorm/migrations/1669403592926-rename_card_status_and_update_app_type_color.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class renameCardStatusAndUpdateAppTypeColor1669403592926
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      ` 
+      UPDATE "card_status"
+      SET "label" = 'App Prelim Done / To Be Assigned to LUP'
```
Comment (dhaselhan) [inline alcs-api/src/providers/typeorm/migrations/1669403592926-rename_card_status_and_update_app_type_color.ts:10]: You could change this to:
App&nbsp;Prelim&nbsp;Done / To&nbsp;Be&nbsp;Assigned&nbsp;to&nbsp;LUP 
so it wraps nicer but up to you

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

File: alcs-frontend/src/app/features/board/board.component.scss (hunk 1 above)
Comment (sandratoh) [reply alcs-frontend/src/app/features/board/board.component.scss:15]: Just tested it and seems like we can remove it from `.column-header`, I'll go ahead and clean that up

File: alcs-api/src/providers/typeorm/migrations/1669403592926-rename_card_status_and_update_app_type_color.ts (hunk 2 above)
Comment (sandratoh) [reply alcs-api/src/providers/typeorm/migrations/1669403592926-rename_card_status_and_update_app_type_color.ts:10]: Good idea!

**Pull request merged**